db/schema.rb

```diff
@@ -31,5 +49,8 @@
     t.index ["amusement_park_id"], name: "index_rides_on_amusement_park_id"
   end
 
+  add_foreign_key "mechanics", "amusement_parks"
+  add_foreign_key "mechanics_rides", "mechanics"
+  add_foreign_key "mechanics_rides", "rides"
   add_foreign_key "rides", "amusement_parks"
 end
```

Comment:
Your database setup looks great. You have your mechanics table, and you have the M2M joins table for a Mechanics and Rides.